app/lib/actions/candlepin/product/content_add.rb

```diff
@@ -2,6 +2,8 @@ module Actions
   module Candlepin
     module Product
       class ContentAdd < Candlepin::Abstract
+        DEFAULT_ENABLEMENT = false
```

Comment:
When you're finished testing the PR, you can change this back to true and then run the rake task (and then remove all the content overrides ;)